### ISSUE TITLE
Firefox 35, 39, 47 and 49; Right file pane is Missing with Ubuntu 14.04 LTS [Solved]

### ISSUE THREAD

**Issue by rwaguy** [Real-Gecko/Filemin#121]
I've used Filemin 2.0.2 with Centos 7-7.2, and it worked well.  However, I recently installed it on an Ubuntu 14.04 LTS server with Virtualmin 5.04, and the right pane where files would normally be listed is missing.  By "missing", I mean the right panel doesn't exist when I use Firebug to inspect the resulting page.

It's not a server or folder permissions problems, because it works as expected in Chromium 31, and I'm reasonably sure that it's not a Firefox plugin problem, because it worked fine on the Centos7 systems, and starting Firefox in Safe Mode doesn't change the result.  Changing the webmin theme also doesn't correct the issue.

It's possible that I'm missing a dependency, although I'm not sure where to check for filemin-specific errors.  (There's nothing obvious in the system or apache logs.)  I've tried (re)installing using the github module, .deb, and outdated webmin module and applied the update, completely removing all traces of Filemin before each installation type, and have gotten the same result.

![screenshot-firefox-47-and-49 0 1](https://cloud.githubusercontent.com/assets/4008601/19412400/63134752-92e3-11e6-9177-4a90340d2ab9.png)
![screenshot-chromium-31](https://cloud.githubusercontent.com/assets/4008601/19412401/631675e4-92e3-11e6-9fc4-e266e93a5468.png)

`````` <body>
<div class="container-fluid">
    <nav id="main-menu" class="navbar navbar-default">
    <ul id="tabs-plus" class="nav nav-tabs" role="tablist">
    <ul id="tabs-control" class="nav nav-tabs ui-sortable" role="tablist">
    <div id="filemin-tree" class="filemin-tree pull-left ui-resizable" style="width: 240px;">
    <div id="tabs-container" class="tab-content pull-left" style="width: 720px;">
    <div id="bottom-panel" class="panel panel-default">
</div>
<ul id="context-menu" class="dropdown-menu">
<ul id="context-menu-search" class="dropdown-menu">
<input id="fileupload" name="files[]" multiple="" type="file">
<div id="chownDialog" class="modal fade">
<div id="chmodDialog" class="modal fade">
<div id="downFromUrlDialog" class="modal fade">
<div id="searchDialog" class="modal fade">
<div id="propertiesDialog" class="modal fade">
<div id="bookmarksDialog" class="modal fade">
<div id="compressDialog" class="modal fade">
</body>```


``````

**Comment by Real-Gecko** [comment]
What happens if you click little button in bottom left corner that hides treeview?
Tried page reload?

**Comment by rwaguy** [comment]
Alright, now I feel foolish.  I saw no treeview button, so I resized the window and caught a glimpse of the missing portions positioned below the edge of the window.  There was no scrollbar, and adjusting the window size merely bumped everything lower, out of view.

However, lowering the font size (Ctrl-mousewheel, or via the menu) now shows everything.

I obviously had the window wider while I was using Centos7.

Thank you for the quick response!
